src/Components/App/App.js

```diff
@@ -32,86 +32,104 @@ class App extends Component {
 
   getActivityData = async (event) => {
     let promise;
-    try{
-      if(event.target.id === 'movies' ) {
-          promise = await getAllMovies();
+    try {
+      if (event.target.id === 'movies') {
+        promise = await getAllMovies();
       }
-      if(event.target.id === 'boardGames' ) {
+      if (event.target.id === 'boardGames') {
         promise = await getAllBoardGames();
       }
-      if(event.target.id === 'cardGames' ) {
+      if (event.target.id === 'cardGames') {
         promise = await getAllCardGames();
       }
-      if(event.target.id === 'music' ) {
-          promise = await getAllMusic();
+      if (event.target.id === 'music') {
+        promise = await getAllMusic();
       }
-      if(event.target.id === 'podcasts' ) {
+      if (event.target.id === 'podcasts') {
         promise = await getAllPodcasts();
       }
-      this.setState({ [event.target.id]: promise})
-      } catch (error){
-        console.log(error)
-      }
+      this.setState({[event.target.id]: promise})
+    } catch (error) {
+      console.log(error)
+    }
   }
-  
+
   updateActivityAnswers = (event) => {
     if (event.target.id !== 'default') {
-      this.setState({ [event.target.id]:  [...this.state[event.target.id], event.target.textContent]})
+      this.setState({[event.target.id]: [...this.state[event.target.id], event.target.textContent]})
     }
   }
 
   setActivities = (activities) => {
     this.setState({activities: [...activities]})
   }
 
-  render() {
+  determineRandomActivity = () => {
+    console.log('yeet')
+    // redirect to temporary loading page while the below logic is run
+    // what activities did the user select?
+    // fetch all activities and place somewhere to be filtered through, here? Result component?
```

Comment:
I believe the fetch is happening in `App.js` so we'd only need to pass the appropriate data as props.  I think maybe we should keep the filtering in `App.js` so that we could keep `Result` as a functional component.